server/app.js

```diff
@@ -262,14 +262,26 @@ app.get('/print/:id', asyncHandler(async (req, res, next)=>{
 //Render the page
 const templateFn = require('./../client/template.js');
 app.use((req, res)=>{
+	// Create configuration object
+	const configuration = {
+		local       : false,
+		environment : config.get('node_env')
+	};
+	// Add local only items to configuration object
+	const localEnvironments = config.get('local_environments');
+	if(localEnvironments.includes(configuration.environment)){
+		configuration.local = true;
+		configuration.secret = config.get('secret');
+	};
 	const props = {
 		version     : require('./../package.json').version,
 		url         : req.originalUrl,
 		brew        : req.brew,
 		brews       : req.brews,
 		googleBrews : req.googleBrews,
 		account     : req.account,
-		enable_v3   : config.get('enable_v3')
+		enable_v3   : config.get('enable_v3'),
+		config      : configuration

```

Comment:
My first thought was anything that gets pulled via `config.get()` - the only exception to this being `enable_v3` as I didn't want to mess with existing functionality.
However this could easily be extended to user configuration; the user's preferred default renderer, theme, etc. However while this change would lay the groundwork for such features, they are well outside the scope of this PR.